### ISSUE TITLE
add alt text to welcome logo

### ISSUE THREAD

**Issue by johrstrom** [OSC/ondemand#2067]
The welcome logo has no alt text. This issue is to add support for internationalized alt text here.

https://github.com/OSC/ondemand/blob/64f6b7deb82168701a7e268a8c00221ffd2230c1/apps/dashboard/app/views/shared/_welcome.html.erb
